loop/helpline_view.py

```diff
@@ -0,0 +1,170 @@
+import time
+import datetime
+import requests
+import unicodecsv as csv
+import xml.etree.ElementTree as xml_parse
+from pytz import timezone
+
+from loop.models import HelplineExpert, HelplineIncoming, HelplineOutgoing, \
+    HelplineCallLog, HelplineSmsLog
+
+from dg.settings import EXOTEL_ID, EXOTEL_TOKEN, EXOTEL_HELPLINE_NUMBER, MEDIA_ROOT
+
+HELPLINE_LOG_FILE = '%s/loop/helpline_log.log'%(MEDIA_ROOT,)
+
+def write_log(logfile,module,log):
+    curr_india_time = datetime.datetime.now(timezone('Asia/Kolkata'))
+    with open(logfile, 'ab') as csvfile:
+        file_write = csv.writer(csvfile, quoting=csv.QUOTE_ALL)
+        file_write.writerow([curr_india_time,module,log])
+
+def save_call_log(call_id,from_number,to_number,call_type,start_time):
+    call_obj = HelplineCallLog(call_id=call_id,from_number=from_number,to_number=to_number,call_type=call_type,start_time=start_time)
+    try:
+        call_obj.save()
+    except Exception as e:
+        # if error then log
+        module = 'save_call_log'
+        write_log(HELPLINE_LOG_FILE,module,str(e))
+
+def save_sms_log(sms_id,from_number,to_number,sms_body,sent_time):
+    sms_obj = HelplineSmsLog(sms_id=sms_id,from_number=from_number,to_number=to_number,sms_body=sms_body,sent_time=sent_time)
```

Comment:
Why do we need to save sms_body in each SMS, Can't we save a pointer to SMS bodies since we have limited SMSes only.